index.d.ts

```diff
@@ -0,0 +1,183 @@
+}
+
+declare class AudioMotionAnalyzer {
+  constructor(container?: HTMLElement, options?: Options);
+
+  get audioCtx(): AudioContext;
+  get canvas(): HTMLCanvasElement;
+  get canvasCtx(): CanvasRenderingContext2D;
+
+  get barSpace(): number;
+  set barSpace(value: number);
+
+  public bgAlpha: number;
+
+  get connectedSources(): AudioNode[];
+
+  get energy(): number;
+
+  get fftSize(): number;
+  set fftSize(value: number);
+
+  public fillAlpha: number;
+
+  get fps(): number;
+
+  get fsHeight(): number;
+  get fsWidth(): number;
+
+  get gradient(): string;
+  set gradient(value: string);
+
+  get height(): number;
+  set height(h: number);
+
+  get width(): number;
+  set width(w: number);
+
+  get isFullscreen(): boolean;
+  get isLedDisplay(): boolean;
+  get isLumiBars(): boolean;
+  get isOctaveBands(): boolean;
+
+  get isOn(): boolean;
+
+  public lineWidth: number;
+
+  get loRes(): boolean;
+  set loRes(value: boolean);
+
+  get lumiBars(): boolean;
+  set lumiBars(value: boolean);
+
+  get maxDecibels(): number;
+  set maxDecibels(value: number);
+
+  get minDecibels(): number;
+  set minDecibels(value: number);
+
+  get maxFreq(): number;
+  set maxFreq(value: number);
+
+  get minFreq(): number;
+  set minFreq(value: number);
+
+  get mode(): number;
+  set mode(value: number);
+
+  public overlay: boolean;
+
+  get peakEnergy(): number;
+  get pixelRatio(): number;
+
+  get radial(): number;
+  set radial(value: number);
+
+  public reflexAlpha: number;
+  public reflexBright: number;
+  public reflexFit: boolean;
+
+  get reflexRatio(): number;
+  set reflexRatio(value: number);
+
+  public showBgColor: boolean;
+  public showFPS: boolean;
+
+  get showLeds(): boolean;
+  set showLeds(value: boolean);
+
+  public showPeaks: boolean;
+  public showScaleX: boolean;
+  public showScaleY: boolean;
+
+  get smoothing(): number;
+  set smoothing(value: number);
+
+  get spinSpeed(): number;
+  set spinSpeed(value: number);
+
+  get splitGradient(): boolean;
+  set splitGradient(value: boolean);
+
+  get stereo(): boolean;
+  set stereo(value: boolean);
+
+  get volume(): number;
+  set volume(value: number);
+
+  static get version(): string;
+
+  public onCanvasDraw: OnCanvasDrawFunction | undefined;
+  public onCanvasResize: OnCanvasResizeFunction | undefined;
+
+  public connectInput(source: HTMLMediaElement): MediaElementAudioSourceNode;
+  public connectInput(source: AudioNode): AudioNode;
+  public connectOutput(node?: AudioNode): void;
+
+  public disconnectInput(node?: AudioNode): void;
+  public disconnectOutput(node?: AudioNode): void;
+
+  public registerGradient(name: string, options: GradientOptions): void;
+
+  public setCanvasSize(width: number, height: number): void;
+  public setFreqRange(minFreq: number, maxFreq: number): void;
+  public setSensitivity(minDecibels: number, maxDecibels: number): void;
+
+  public toggleFullscreen(): void;
```

Comment:
missing [`setOptions()`](https://github.com/hvianna/audioMotion-analyzer#setoptions-options-) and [`toggleAnalyzer()`](https://github.com/hvianna/audioMotion-analyzer#toggleanalyzer-boolean-) methods

Reply:
Not sure how I missed those two, my bad. Updated now.